public/app/features/plugins/partials/ds_http_settings.html

```diff
@@ -94,6 +95,24 @@ <h6>Basic Auth Details</h6>
   </div>
 </div>
 
+<div class="gf-form-group" ng-if="current.addCustomHeaders">
```

Comment:
Its impossible to add new headers since the add button is placed within the repeater. If the list is empty nothing shows up.